povm_toolbox/library/metadata/rpm_metadata.py

```diff
@@ -26,7 +26,7 @@ class RPMMetadata(POVMMetadata):
     pvm_keys: np.ndarray
     """The keys which associate a specific result sample with the corresponding RPM parameters.
 
-    Shape of `pvm_keys` is assumed to be ``(*pv.shape, num_batches, n_qubit)``,
+    Shape of `pvm_keys` is assumed to be ``(*pv.shape, num_batches, num_qubit)``,
```

Comment:
Small detail but here shouldn't it be `num_qubits` with an "s" ?

Reply:
Nice job spotting this! :+1: